dist/scss/wds-mixins/_wds-hide-on-breakpoint.scss

```diff
@@ -0,0 +1,50 @@
+@mixin wds-hide-on-breakpoint($breakpoints) {
+	// only
+	@if index($breakpoints, small) {
+		@media #{$wds-breakpoint-small-only} { display: none; }
```

Comment:
You may consider using `!important` for this mixin. That would assure it'll always work ;)